source/agora/consensus/Fee.d

```diff
@@ -370,9 +370,9 @@ public class FeeManager
             {
                 auto new_fee = this.accumulated_fees[stake.output.address]
                     .toString();
-                this.db.execute("INSERT INTO accumulated_fees (public_key, fee)"
-                    ~ " VALUES (?,?) ON CONFLICT(public_key) DO UPDATE SET fee = ?",
-                    stake.output.address.toString(), new_fee, new_fee);
+                this.db.execute("INSERT OR REPLACE INTO accumulated_fees " ~
```

Comment:
You can use `REPLACE INTO` instead of `INSERT OR REPLACE INTO`. The statement is already used in the `EnrollmentManager`.